managers/tests/test_unit_managers_apiview.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+
+from rest_framework.reverse import reverse
+from rest_framework.test import APIRequestFactory
+
+from django.test import TestCase
+
+from users.models import CustomUser
+from managers import views
+from managers.models import Project
+
+
+class ProjectTest(TestCase):
+    def setUp(self):
+        self.user = CustomUser(
+            email="testuser@codepoets.it",
+            first_name='John',
+            last_name='Doe',
+            country='PL',
+            user_type=CustomUser.UserType.ADMIN.name,
+        )
+        self.user.password = 'newuserpasswd'
+        self.user.set_password(self.user.password)
+        self.user.full_clean()
+        self.user.save()
+
+        self.project = Project(
+            name='Example Project',
+            start_date=datetime.datetime.now().date() - datetime.timedelta(days=30),
+            stop_date=datetime.datetime.now().date(),
+            terminated=False,
+        )
+        self.project.full_clean()
+        self.project.save()
+        self.project.managers.add(self.user)
+        self.project.members.add(self.user)
+        self.url = [
```

Comment:
You are creating a list here, but in test you always refer to it as `self.url[0]`. This can be simplified to be just simple string, and then it can accessed directly in test like `self.url`.

Reply:
@rwrzesien I did it this way, because after adding next features (detail, update and so on) this will change into array of url (to store all tested views). I thought that this way of prediction would be good ;)

Reply:
@MartynaAnnaGottschling I get this, but those should be separated, independent classes. You can always overwrite this attribute in subclasses and use the same `self.url` in all of them, and it will have different value in each subclass. Or do it like @kbeker proposed.

Reply:
Done